Form/Extension/DisableCSRFExtension.php

```diff
@@ -68,7 +68,7 @@ public function getExtendedType()
             ;
     }
 
-    public static function getExtendedTypes()
+    public static function getExtendedTypes() : iterable
```

Comment:
I did this change, I believe it is an acceptable bc break as it is not a class meant to be extended imo (should it be made `@final` by the way ?).

Reply:
LGTM